app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -195,6 +195,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 * Whether items shown in the screensaver are required to have an age rating set.
 		 */
 		var screensaverAgeRatingRequired = booleanPreference("screensaver_agerating_required", true)
+
+		/**
+		 * Whether to prefer extension renderers (i.e. FFmpeg) to core ones.
+		 */
+		var preferExtensionRenderers = booleanPreference("prefer_extension_renderers", defaultValue = false)
```

Comment:
```suggestion
		var preferExoPlayerFfmpeg = booleanPreference("exoplayere_prefer_ffmpeg", false)
```

Reply:
Also move it up to after refreshRateSwitchingBehavior so it's grouped with the other playback preferences.

Reply:
done